BUILD.gn

```diff
@@ -162,17 +162,38 @@ if (!is_mac) {
 
     sources = []
     foreach(locale, locales) {
-      # print (locale)
-      if (is_mac) {
-        sources += [ "$root_gen_dir/repack/locales/$locale.pak" ]
-      } else {
-        sources += [ "$root_out_dir/locales/$locale.pak" ]
-      }
+      sources += [ "$root_out_dir/locales/$locale.pak" ]
     }
     outputs = [
       "$brave_dist_dir/locales/{{source_file_part}}",
     ]
   }
+
+  group("brave_extension_locale_dist_resources") {
```

Comment:
Isn't this already done in `components/brave_rewards/resources/extension/BUILD.gn`?